public/assets/css/style.css

```diff
@@ -383,12 +383,14 @@ tfoot {
 .card-fighter {
     background-color: var(--white);
     border: 2px solid var(--black);
-    padding: 1rem;
+    padding: 0 2rem;
 }
 .card-img {
     background-color:var(--primaryColorAlter);
     border: 3px solid var(--black);
-    padding: 3rem;
+    height: 30vh;
```

Comment:
this can lead to unstable behaviour: when your window grows, the image will grow as well. prefer rem